core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -168,6 +236,50 @@ protected void doStop() throws Exception {
     return server;
   }
 
+  private static String[] getList(String original) {
```

Comment:
Don't we have a `LIST` type in ConfigDef? Can we use that instead and get rid of this custom parsing code?